src/components/BookList.js

```diff
@@ -1,9 +1,18 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import Books from './Books';
 import Form from './Form';
 import '../Assests/bookList.css';
+import { removeBook } from '../redux/books/booksSlice';
 
 function BookList() {
+  const { books } = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+
+  const handleDelete = (id) => {
+    dispatch(removeBook(id));
+  };
+  // console.log('books', books[0].itemId);
```

Comment:
**You did great job!**

- [ ] However, your code needs some cleanup job! To keep your code clean please remove this commented code from your file.